tests/migrations/test_writer.py

```diff
@@ -252,9 +252,10 @@ def test_serialize_constants(self):
         self.assertSerializedEqual(False)
 
     def test_trying_to_serialize_a_lambda_in_field_argument_should_raise_helpful_message(self):
```

Comment:
A test name over 79 characters seems too verbose to me.

Maybe: `test_serialize_field_with_lambda` ?

Reply:
Thanks for the suggestion. I will update it.